src/mlpack/tests/main_tests/hmm_loglik_test.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file hmm_loglik_test.cpp
+ * @author Daivik Nema
+ *
+ * Test mlpackMain() of hmm_loglik_main.cpp
+ */
+#include <string>
+
+#define BINDING_TYPE BINDING_TYPE_TEST
+static const std::string testName = "HMMLoglik";
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include "test_helper.hpp"
+#include <mlpack/methods/hmm/hmm_model.hpp>
+#include <mlpack/methods/hmm/hmm.hpp>
+#include <mlpack/methods/hmm/hmm_loglik_main.cpp>
+
+#include <boost/test/unit_test.hpp>
+#include "../test_tools.hpp"
+
+#include "hmm_test_utils.hpp"
+
+using namespace mlpack;
+
+struct HMMLoglikTestFixture
+{
+ public:
+  HMMLoglikTestFixture()
+  {
+    // Cache in the options for this program.
+    CLI::RestoreSettings(testName);
+  }
+
+  ~HMMLoglikTestFixture()
+  {
+    // Clear the settings.
+    bindings::tests::CleanMemory();
+    CLI::ClearSettings();
+  }
+};
+
+BOOST_FIXTURE_TEST_SUITE(HMMLoglikMainTest, HMMLoglikTestFixture);
+
+BOOST_AUTO_TEST_CASE(HMMLoglikOutputNegativeTest)
+{
+  // Create an HMMModel
```

Comment:
I know this is picky, but can you make sure that comments are complete sentences that end with periods?
https://github.com/mlpack/mlpack/wiki/DesignGuidelines#comments